.github/workflows/publish_docker_images_cron.yml

```diff
@@ -20,15 +20,18 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone --depth=1 --branch update_properties_fixes https://github.com/citusdata/tools.git tools
+        run: git clone --depth=1 --branch develop https://github.com/citusdata/tools.git tools
 
       - name: Install package dependencies
         run: sudo apt install libcurl4-openssl-dev libssl-dev
 
       - name: Install python requirements
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
+      - name: DockerHub Login
+        run: docker login -u ${{ secrets.DOCKERHUB_USER_NAME }} -p ${{ secrets.DOCKERHUB_PASSWORD }}
+
       - name: Build and publish docker images
         run: |
           python -m  tools.packaging_automation.publish_docker  --pipeline_trigger_type "${GITHUB_EVENT_NAME}"\
-           --exec_path "$(pwd)" --image_type ${{ matrix.image_type }}
+          --exec_path "$(pwd)" --image_type ${{ matrix.image_type }}
```

Comment:
I think we should remove `--exec-path`   param from all the scripts and prefer the github actions based approach that you shared in other pr, but this is not related with this pr.
Can you open an issue for that in tools repo ?

Reply:
Discussed in our private chat and seems that I misunderstood the usage of this param.

However, we still need to introduce this to README.

Reply:
Added in readme in tools repo